frontend/src/components/UserCreate.vue

```diff
@@ -2,9 +2,7 @@
 <template>
   <div id="user">
     <h1>Create New User</h1>
-    <user-details ref="userDetails">
-      <v-btn id="submit-button" class="secondary" medium @click="createUser()">Create User</v-btn>
-    </user-details>
+    <user-details ref="userDetails" update-or-create="Create" @submit-user-updates="createUser"></user-details>
```

Comment:
The user to update is now emitted from the child component (user-details) rather than pulled from the vuex store.

Reply:
Is the `update-or-create` attribute used? I don't see anything that references it.